crypto/crypto.go

```diff
@@ -75,7 +74,7 @@ func (s cryptoSource) Int63() int64 {
 func (s cryptoSource) Uint64() (v uint64) {
 	err := binary.Read(crand.Reader, binary.BigEndian, &v)
 	if err != nil {
-		util.Check(err, "generate random verification code")
+		logger.Fatal("failed to generate random verification code: %w", err)
```

Comment:
while I think it'd be very rare for an error to occur from the generator, I think we should return an error rather than shutting down the entire server.

The error needs to be properly handle the error in the case of "we couldn't generate the verification code."

Reply:
Yeah, I see that now with `util.Check` it was doing log fatal in there. I don't know if @cblgh has preference on splitting that out into a separate PR or not.